sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -12,7 +12,7 @@ The Azure Event Hubs client library allows for publishing and consuming of Azure
 
 - Receive events from one or more publishers, transform them to better meet the needs of your ecosystem, then publish the transformed events to a new stream for consumers to observe.
 
-[Source code](.) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.EventHubs/) | [API reference documentation](https://azure.github.io/azure-sdk-for-net/api/Azure.Messaging.EventHubs.html) | [Product documentation](https://docs.microsoft.com/en-us/azure/event-hubs/)
+[Source code](.) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.EventHubs/) | [API reference documentation](https://azure.github.io/azure-sdk-for-net/eventhub.html) | [Product documentation](https://docs.microsoft.com/en-us/azure/event-hubs/)
```

Comment:
The product name is `Event Hubs`.   Can we please add the "s" to the page to reflect that?

Reply:
That just uses the name of the service in the repo which is currently `eventhub`

Reply:
Forgive me, but I'm not sure if that indicates that this cannot be changed without effort because it's automatically populating or if it would just violate a convention that is currently in place.  Would you please be so kind as to clarify?

Reply:
Well, We have just been thinking of the best way to go and decided the general service page will be best. We could still link to the specific package/version if you wish.

cc @weshaggard @AlexGhiondea

Reply:
I've no issue with the general service page approach.   I'd like to understand if the name of said page is automatically generated based on the path in the repository or if it can easily be adjusted without the need to change the path.   

In the first case, I don't feel strongly enough about it to do the path change, due to the use of the wrong product name elsewhere and because of the disruption it would cause.   On the other hand, if that page name is not generated from the path and can easily be changed, I believe it would be a good idea to use the correct service name.

Reply:
The page is generated from naming in the repo

Reply:
Fair enough.   I withdraw my request.

Reply:
@scbedd correct me if I'm wrong but I though there was a mapping we had somewhere that we could map the eventhub folder to a nicer name "Event Hubs". @chidozieononiwu  is that not yet committed to the .NET repo?

Reply:
@weshaggard Yes, Karishma is indeed doing this. My understanding was that @chidozieononiwu was also executing on that. Whether or not it's committed yet is a different story.

Reply:
Oh, I don't have that done yet. But I will get to work on it.